src/python/WMCore/MicroService/MSTransferor/MSTransferor.py

```diff
@@ -195,6 +202,13 @@ def execute(self, reqStatus):
             self.logger.info("%d requests information completely processed.", len(reqResults))
 
             for wflow in reqResults:
+                # perform site list updates
+                errors = self._updateSites(wflow)
+                if len(errors) == 0:
```

Comment:
In practice, you are overwriting this metric with the very last workflow outcome.
Instead, the way it has been used so far is to provide a summary of the microservice execution cycle.

Said that, my suggestion would be to define it to an integer number saying how many workflows (count) have been re-placed.